src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -7,28 +7,28 @@
     private static final int MAX_VALUE = 127;
     private static final int NUMBER_OF_FIGURES = 5;
     private Random random = new Random();
-    private int firstRandomNum = random.nextInt((MAX_VALUE - MIN_VALUE) + 1) + MIN_VALUE;
-    private int secondRandomNum = random.nextInt((MAX_VALUE - MIN_VALUE) + 1) + MIN_VALUE
-            + random.nextInt(MAX_VALUE);
-    private int thirdRandomNum = firstRandomNum + secondRandomNum;
+
+    private int getRandomNum() {
```

Comment:
private methods should be listed after public

Reply:
fixed